main.go

```diff
@@ -90,6 +92,7 @@ func main() {
 		log.Fatal(err)
 	}
 
+	log := zapr.NewLogger(logger)
```

Comment:
- shadowing std lib here, can this be `logger` and o/p of `zapConfig.Build()` be `zapLogger`?